labs/firepower-restapi-105/1.md

```diff
@@ -1,38 +1,29 @@
 # FMC - REST - API - 105
 
-## Lab Objective
+## Lab objective
 
-Basic understanding of debugging the FMC REST API errors
+This lab gives a basic understanding of debugging the FMC REST API errors.
 
-Completion time: 10 minutes
 
 ## Prerequisites
-Basic understanding of REST principles
+It's best if you have a basic understanding of REST principles.
 
 
-## Learning Objective
-After Completing this module you will be able to:
-Get basic understanding of debugging FMC REST APIs.
+## Learning objective
+After Completing this module you will be able to get basic understanding of debugging FMC REST APIs.
+
 
 ## Problem: HTTP 404 for a request
-Most likely the resource URL used in the REST Client had an error. Use browser to access the FMC API explorer, and verify that the relative url used in your REST client script matches the one in the FMC API explorer. If it doesn’t, fix the url in the REST client script.
-if you are using devnet sandbox  https://fmcrestapisandbox.cisco.com/api/api-explorer/
-or incase it's your own setup
-https://hostnameorIPaddressofyourFMC/api/api-explorer
-If you still get a 404, you have to login to the FMC GUI and verify that the resource corresponding to the id in the url exists on the FMC.
+Most likely the resource URL used in the REST Client had an error. Use the browser to access the FMC API explorer, and verify that the relative URL used in your REST client script matches the one in the FMC API explorer. If it doesn’t, fix the URL in the REST client script. If you are using [DevNet Sandbox](https://fmcrestapisandbox.cisco.com/api/api-explorer/), or if it's your [own setup](https://hostnameorIPaddressofyourFMC/api/api-explorer), and if you still get a 404, you have to login to the FMC GUI and verify that the resource corresponding to the ID in the URL exists on the FMC.
```

Comment:
Use API Explorer to be consistent with prior files.

Reply:
Since "https://hostnameorIPaddressofyourFMC/api/api-explorer" is a made-up URL and won't resolve correctly, don't put it in Markdown link syntax, keep it on its own line.

Reply:
Okay, I undid the login, and moved it to it's own line, but it's still showing up as a link (at least in the editor). Just so you know.